_VERSION_MIGRATION/migrate-to-next.py

```diff
@@ -102,7 +102,7 @@ def get_spec(config):
         containers = config['containers']
 
     return {
-        'terminationGracePeriod': 100,
+        'terminationGracePeriod': "100s",
```

Comment:
Hey @manuellysuzik what do you think to try to get the value from the v9 configuration and if it is not present, set this default timeout as `100s`?

Reply:
sure! Working on it